### PR TITLE
Fixed share indicator delay

### LINKED ISSUE #2973: Share indicators appear delayed
### Steps

1. Create a folder "test"
1. Share "test" with link or a collaborator
1. Refresh the page and observe closely the progress bar and the indicators

### Expected result
Indicator appears immediately after the progress bar disappears.

### Actual result
First the progress bar disappears, the file list appears, and then after a second the share indicator appears

### Version
v0.3.0

This is a regression as it used to work when I worked with sharing indicators.
The code that loads the share indicators was designed to happen while the progress bar is still shown. In other words, the progress bar should show as long as neither file list nor indicators have finished loading.

Could this be related to your changes @LukasHirt ?

cc @kulmann if you want to look into this

### PR DESCRIPTION
**Pull request by Julian1998**
## Description

Load the share (default) indicators in the general file list component first and pass them down. 

## Related Issue

- Fixes #2973 

## Motivation and Context

Bugfix

## How Has This Been Tested?

- Using Chrome

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by Julian1998** [comment]
@LukasHirt Is the changelog correct? It's my first :)

File: apps/files/src/components/FilesLists/Indicators.vue [hunk 1]
```diff
@@ -20,24 +32,18 @@
...
 export default {
-  name: 'StatusIndicators',
-
-  components: {
-    DefaultIndicators
-  },
+  name: 'Indicators',
 
   props: {
     // FIXME: Find a way to pass item into dynamic component as a prop without mutation error
```
Comment (LukasHirt) [inline apps/files/src/components/FilesLists/Indicators.vue]: Move the TODO above `item` prop

File: apps/files/src/components/FilesLists/Indicators.vue [hunk 2]
```diff
@@ -1,12 +1,24 @@
 <template>
   <span>
-    <DefaultIndicators
-      v-if="displayDefaultIndicators"
-      :item="item"
-      :parentPath="parentPath"
-      :class="{ 'uk-margin-xsmall-right' : customIndicators }"
-      @click="openSidebar"
-    />
+    <div v-show="displayDefaultIndicators" :class="{ 'uk-margin-xsmall-right' : customIndicators }"
```
Comment (LukasHirt) [inline apps/files/src/components/FilesLists/Indicators.vue]: Use `v-if` pls

File: apps/files/src/components/AllFilesList.vue [hunk 3]
```diff
@@ -58,8 +58,8 @@
           class="uk-margin-small-left"
         />
       </div>
-      <div v-if="!$_isFavoritesList" class="uk-flex uk-flex-middle uk-flex-right">
-        <StatusIndicators :item="item" :parentPath="currentFolder.path" @click="$_openSideBar" />
+      <div v-show="!$_isFavoritesList" class="uk-flex uk-flex-middle uk-flex-right">
```
Comment (LukasHirt) [inline apps/files/src/components/AllFilesList.vue]: Does it still appear delayed if you use `v-if`? If not pls use that instead.

**Pull request merged**